src/include/ipc4/base_fw.h

```diff
@@ -728,4 +728,7 @@ struct schedulers_info {
 
 struct ipc4_system_time_info *basefw_get_system_time_info(void);
 
+/* Specifies I2S IPC4 version for PTL platform */
+static const uint32_t I2S_VER_30_PTL       = 0x40000;
+
```

Comment:
This would ideally be put in some Intel-specific file, but as we have no such file now, base_fw.h is probably the best fit. But if we get more of these, we need to add base_fw_intel.h and have these there (and also if there are any build-time dependencies).